frontend/src/api/k8s/accelerators.ts

```diff
@@ -2,7 +2,10 @@ import { k8sListResource } from '@openshift/dynamic-plugin-sdk-utils';
 import { AcceleratorKind } from '~/k8sTypes';
 import { AcceleratorModel } from '~/api/models';
 
-export const listAccelerators = async (): Promise<AcceleratorKind[]> =>
+export const listAccelerators = async (namespace?: string): Promise<AcceleratorKind[]> =>
```

Comment:
This should probably be required -- a wide fetch like that is not something I think we should lean towards as an option -- it's not ideal to fetch so wide "accidentally" -- it's a cost to fetch across all namespaces.